lib/iris/tests/unit/coords/test_Coord.py

```diff
@@ -368,5 +368,21 @@ def test_writable_points(self):
             coord2.bounds[:] = 0
 
 
+class Test_AuxCoord__getitem__(tests.IrisTest):
+    def test_partial_slice_data_copy(self):
+        original_coord = AuxCoord([1., 2., 3.])
+        sub_coord = original_coord[:1]
+        original_values = sub_coord.points.copy()
```

Comment:
I've no problem with the code of the tests :+1: However, I found the variable names a little confusing as you've sort of overloaded the word 'original'.

It refers to both:

* the coord you start with ("`original_coord`"), and
* the initial values of the indexed coord ("`original_values`").

Could you perhaps rename `original_values` to something like `initial_values` to make it clearer that there's no direct connection between it and `original_coord`?